tests/integration/rdb.tcl

```diff
@@ -114,13 +114,37 @@ start_server [list overrides [list "dir" $server_path] keep_persistence true] {
     r del stream
 }
 
+package require Tclx
```

Comment:
Let's add some comment as a heading for this umask test code block, since it's outside of a regular test block.